tests/unit/client/feedback/schemas/test_questions.py

```diff
@@ -73,6 +74,30 @@ def test_label_question_errors(
         LabelQuestion(**schema_kwargs)
 
 
+@pytest.mark.parametrize(
+    "schema_kwargs, expected_warning, expected_warning_message",
+    [
+        (
+            {"name": "a", "description": "a", "required": True, "labels": ["a", "b", "c"], "visible_labels": 4},
+            UserWarning,
+            "\`visible_labels=4\` is greater than the total number of labels \(3\), so it will be set to \`3\`.",
+        ),
+        (
+            {"name": "a", "description": "a", "required": True, "labels": ["a", "b"], "visible_labels": 3},
+            UserWarning,
+            "\`labels=\['a', 'b'\]\` has less than 3 labels, so \`visible_labels\` will be set to \`None\`, which means that all the labels will be visible.",
+        ),
+    ],
+)
+def test_label_question_warnings(
```

Comment:
We should add a test using the default `20` value and options with fewer elements.